src/Meter.cpp

```diff
@@ -143,9 +147,11 @@ void Meter::onTick()
         emit onComputedVolumeMeasurements(mValues);
```

Comment:
Another thought/area of concern was how `mValues` gets passed here but it looks like QT has a copy-on-write paradigm built in? https://doc.qt.io/qt-5/implicit-sharing.html#implicit-sharing-in-detail